example.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<body>
+<h1>Publish Subscribe on Encrypted channel example</h1>
+<button id="publish">Publish encrypted messages</button>
+<br><br>
+<textarea id="subscriptions" style="width: 100%" rows="15"></textarea>
+</body>
+
+<!-- Include the latest Ably Library  -->
+<script src="https://cdn.ably.io/lib/ably.min.js"></script>
```

Comment:
That is not the latest ably library, that's 0.8. The latest is at `ably.min-1.js`. 
(Did you copy this from another tutorial? If so, please let us know so @tomczoink or @Srushtika can fix it)

Reply:
Yes I believe I used https://github.com/ably/tutorials/blob/history-javascript/example.html as a reference

Reply:
Thanks - @tomczoink could you make sure that tutorial is updated, and check for any others that use `ably.min.js` or `ably.js`?